components/datatypes/productlistitem.schema.json

```diff
@@ -94,6 +94,14 @@
           "description": "The difference between the regular price and the special price for the product.",
           "meta:titleId": "productlistitem##xdm:discountAmount##title##25051",
           "meta:descriptionId": "productlistitem##xdm:discountAmount##description##67611"
+        },
+        "xdm:categoryData": {
```

Comment:
can we rename this field to productCategory ? The final fields will look like below then:
category.ID
category.:name
category.path

categoryData.ID does not looks good semantically.

Reply:
@mjasrotia thanks for the review!!

i renamed to `productCategories` since it's an array. is that ok or do you prefer `productCategory` still?

Reply:
@aniham , the plural name is much better but I realized that the fully qualified for these categories would not have this object name in that as its is an array of objects here. What I would like to suggest is to rename the datatype properties by adding a prefix of "category" like categoryID, categoryName, categoryPath.

This way, your payload should look like this:

`productCategories: [
  {
    categoryID:"C100",
    categoryName: "shoes".
    categoryPath: "Somepath"
  },
    {
    categoryID:"C200",
    categoryName: "Clothes".
    categoryPath: "Somepath"
  },
    {
    categoryID:"400",
    categoryName: "Houseware".
    categoryPath: "Somepath"
  }
]`

Reply:
✅